src/modules/systemlib/param/param.c

```diff
@@ -1140,14 +1139,14 @@ param_export(int fd, bool only_unsaved)
 out:
 
 	if (result == 0) {
-		result = bson_encoder_fini(&encoder); // this will call fsync
+		result = bson_encoder_fini(&encoder);
 
 		// write and finish
 		if ((result != 0) || write(fd, encoder.buf, encoder.bufpos) != encoder.bufpos) {
 			PX4_ERR("param write error");
 
 		} else {
-			px4_fsync(fd);
+			fsync(fd);
```

Comment:
I'm not sure why this was done. https://github.com/PX4/Firmware/blob/master/src/modules/systemlib/bson/tinybson.c#L59

Reply:
This goes back to 9f3bf8e9f4b8f5cde42786211280628168246f99